### ISSUE TITLE
LabelsRotation and LayoutTransform do not rotate chart labels and title (180° or over)

### ISSUE THREAD

**Issue by NejibChennoufi** [beto-rodriguez/LiveCharts2#445]
**Describe the bug**
LabelsRotation and LayoutTransform do not rotate labels and title (180° or over). Let say the label is vertical facing left, I want to flip it  vertical facing right.

**To Reproduce**
Just binding livechart axis LabelsRotation to an angle that is 180° or greater.
Same thing for livechart axis title.

**Expected behavior**
I thought I can rotate chart labels and title by any angle that I precise.

**Screenshots**
![image](https://user-images.githubusercontent.com/34676894/165048897-d8c49d19-c3e3-49b5-bcba-099c72c83619.png)

![image](https://user-images.githubusercontent.com/34676894/165050099-1e267124-c67a-4d9b-aa67-b1f9f756e1e5.png)

**Desktop (please complete the following information):**
 - Window's 10
 - WPF, .Net 4.7.2
 - Live-Charts Geared version 1 (licensed)

**Additional context**
I have posted my request https://github.com/Live-Charts/Live-Charts/issues but got no answers which was frustrating especially for paid license. Now, I understand I wasn't targeting the correct forum.
Your help is highly appreciated.

**Comment by beto-rodriguez** [comment]
The library manages the rotation and prioritizes readability, when the rotation is 180 or greater then the library inverts the direction of the label, so the users are able to read them, this is what most of the users need and that is why the library behaves like this.

 Ad a work around for your case, you can try to set the rotation to any angle before 180,  please try setting the rotation to 179.9 and see if that works for you.

**Comment by NejibChennoufi** [comment]
Appreciate your feedback.
Unfortunately, I tried all possibilities but it seems even beyond 90 degrees doesn't work.
In my humble opinion, I think it is simpler to default the values but not to force them.

I start with a grid of rows. Each row implements a chart. The user can rotate the grid by 90degrees to flip rows to columns so the charts become stacked horizontally instead of vertically. I tried with stackpanel, it didn't work for me but I will try to figure out something.

Thank you.
